apps/tlon-mobile/src/fixtures/Channel.fixture.tsx

```diff
@@ -514,4 +593,17 @@ export default {
   notebook: <NotebookChannelFixture />,
   chatWithImage: <ChannelFixtureWithImage />,
   negotiationMismatch: <ChannelFixture negotiationMatch={false} />,
+  customChannel: (
+    <ConfigurableChannelFixture
+      initialContentConfiguration={{
+        ...ChannelContentConfiguration.defaultConfiguration(),
+        defaultPostCollectionRenderer: {
+          id: CollectionRendererId.boardroom,
+          configuration: {
+            showAuthors: true,
+          },
+        },
+      }}
+    />
```

Comment:
I added this to test presets of configurations as fixtures – you can just pass an initial config here, and then you can tap the floating `Toggle configurator` button in the fixture to open the config bar. (I had a notes app example here, but it doesn't work because of navigation)